db/migrate/20191211160017_create_dfe_sign_in_users.rb

```diff
@@ -0,0 +1,13 @@
+class CreateDfeSignInUsers < ActiveRecord::Migration[6.0]
+  def change
+    create_table :dfe_sign_in_users, id: :uuid do |t|
+      t.string :dfe_sign_in_id
```

Comment:
We'll want to make this unique (as it should always be unique) and also indexed (because it's the column we'll be performing the SELECT on when we look up a user record)